appveyor.yml

```diff
@@ -88,6 +88,8 @@ install:
     - cmd: conda install --yes --quiet obvious-ci conda-build-all
     - cmd: conda install --yes --quiet conda-forge-build-setup
     - cmd: run_conda_forge_build_setup
+    # current workaround for issue #1130 in conda-build-all
+    - cmd: conda install --yes --quiet conda-build=1.21.5
```

Comment:
I believe this was fixed b/c I no longer see that in #1133 (which got latest `conda-build`)

Let's try removing this and checking how it goes here.